dnf5daemon-client/wrappers/dbus_transaction_package_wrapper.hpp

```diff
@@ -37,7 +37,7 @@ class DbusTransactionPackageWrapper {
         : package(std::get<1>(dti)),
           action(static_cast<libdnf::transaction::TransactionItemAction>(std::get<0>(dti))),
           // TODO(lukash) reason needs to be added to dbus
-          reason(libdnf::transaction::TransactionItemReason::UNKNOWN) {}
+          reason(libdnf::transaction::TransactionItemReason::DEPENDENCY) {}
```

Comment:
I believe that default value must be user installed. It is safe default.

Reply:
Removal of unknown state without providing code that will handle such a situation sounds not right to me. When will you introduce the logic that will handle that kind of situation?

Reply:
> I believe that default value must be user installed. It is a safe default.

This is just a placeholder until the reason is correctly sent over dbus.

> Removal of unknown state without providing code that will handle such a situation sounds not right to me. When will you introduce the logic that will handle that kind of situation?

I don't know what situation you're talking about.